smokes/e2e/buildsnavigation.scenarios.coffee

```diff
@@ -0,0 +1,67 @@
+# coffee script
+# test goal: checks the capability to navigate in a dedicated build
+# to use previous and next link
+
+
+forcePage = require('./force.coffee')
+builderPage = require('./builder.coffee')
+
+describe('', () ->
+    force = null
+    builder = null
+
+    beforeEach(() ->
+        builder = new builderPage('runtest', 'force')
+        force =  new forcePage()
+        builder.goDefault()
+    )
+
+    describe 'previousnextlink', () ->
+        it 'should navigate in the builds history by using the previous next links', () ->
+
+            # Build #1
+            builder.go()
+            builder.goForce()
+            force.getStartButton().click()
+            browser.waitForAngular()
+            browser.sleep(4000)
```

Comment:
need this time out to be sure that the build is really started and to avoid to group the request
this time out will be reduced to 2000
second solution: to add a parameter which forbids the request group capability